styles/variables-gallium.less

```diff
@@ -150,6 +150,12 @@
 
 // SliderButton
 // ---------------------------------------
+@agate-sliderbutton-border-radius: 3px;
+@agate-sliderbutton-border-width: 12px;
+@agate-sliderbutton-bg-clip: padding-box;
+@agate-sliderbutton-knob-border-radius: 6px;
+@agate-sliderbutton-knob-border-width: 3px;
+@agate-sliderbutton-knob-transform: scale(1.07, 1.4);
```

Comment:
What do you think about having a smaller scale like `scale(1.04, 1.4)`, and also add a top margin as well? So that the knob shadow won't be clipped.
<img width="219" alt="sliderbuttonclip" src="https://user-images.githubusercontent.com/8940009/72563051-b2bdfc80-3861-11ea-8ad8-332144747a36.png">